### PR TITLE
test(wdio): add a browser matrix for our wdio tests

### PR DESCRIPTION
**Pull request by alicewriteswrongs**
This refactors a bit how we control what browsers are used in our webdriverio tests. Instead of it being chrome or all browsers, where all browsers are activated by setting the `CI` environment variable, this adds a check for a `BROWSER` environment variable. How this works is that the webdriverio configuration file reads the environmen variable and if it's one of `FIREFOX`, `CHROME`, or `EDGE` it will set the tests to use that browser. If it's _not_ set (or set to an unrecognized value) we'll default to using chrome (as we do now), and if it's set to `ALL` it will run all of the browsers.

This also makes it easier for us to separate the test runs in separate browsers into separate jobs, so a failure only on Edge will show up more easily (and we'll get some more parallelism).

## What is the current behavior?

You need to edit a file if you want to run tests in e.g. just Edge locally.

## Does this introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

**Review by rwaskiewicz — APPROVED** [review]
LGTM 👍 I'll defer to @christian-bromann in the event he thinks there's another way we could be doing this, but in general I'm in favor of this change.

We'll have to rename the checks when this is ready to merge, LMK when it's time and we'll do the dance of disabling the current one, merging this, and updating the names of the checks in GH's settings

File: .github/workflows/test-wdio.yml
Comment (christian-bromann) [inline .github/workflows/test-wdio.yml:45]: Alternatively:
```suggestion
        run: npm run test.wdio
        env:
          BROWSER: ${{ matrix.browser }}
```

**Review by christian-bromann — APPROVED** [review]
LGTM 👍

File: .github/workflows/test-wdio.yml
Comment (alicewriteswrongs) [reply .github/workflows/test-wdio.yml:45]: true! a little less implicit that way

**Pull request merged**
